context/metadata.ts

```diff
@@ -16,6 +17,11 @@ const dispatch: { [key: string]: version } = {
   "0.1.1": "0.1.1",
 };
 
+const VERSION_HASH: { [k: string]: version } = {
```

Comment:
can you add some comments to explain where this hash coming from and why not have version `< 0.0.10` ?